content/docs-v1/figure-images.md

```diff
@@ -141,23 +115,39 @@ Quire supports video embeds from either YouTube (`media_type: youtube`) or Vimeo
   media_type: youtube
 ```
 
-{{< q-class "box tip" >}}
+The `src` image provided in this example is a frame from the video and will be used in place of the video in the PDF and EPUB versions of your publication. In Quire this is referred to as a fallback. Along with the fallback image, Quire will also automatically append a link to the video following the caption.
 
-- The `src` image provided in this example is a frame from the video and will be used in place of the video in the PDF and EPUB versions of your publication. In Quire this is referred to as a fallback. Along with the fallback image, Quire will also automatically append a link to the video following the caption.
+{{< q-class "box tip" >}}
 
-  Like the [image labels](#labeling-figure-images) this is controlled in the project’s `config.yaml` file with `videoFigureFallbackText: true`, `videoFigureFallbackTextBefore: "Watch the video at "` and `videoFigureFallbackTextAfter: "."`.
+- Note that on YouTube, videos can be filed as “Unlisted” and this will let you embed the video, but will not include the video on your channel page, or in YouTube’s general search engine.
 
 {{< /q-class >}}
 
-{{< q-class "box tip" >}}
+## Add Soundcloud Files
 
-- Note that on YouTube, videos can be filed as “Unlisted” and this will let you embed the video, but will not include the video on your channel page, or in YouTube’s general search engine.
+Embedding Soundcloud files follows a similar process to embedding videos. The YAML would include `media_type: soundcloud`. For the `media_id`, go to the Soundcloud file you'd like to feature and click the "share" button. Next, click the "embed" button which will open a long line of code. After "src=" you will see a url that includes string of nine numbers. This is your `media_id`.
 
-{{< /q-class >}}
+{{< q-figure id="cva_podcast" >}}
+
+For example, if you wanted to include the audio clip above in a Quire project, you would click "share" and then "embed". Then look for the following url embedded in the code:
+
+```md
+src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/809104705
+```
+
+You would then grab `809104705` and add that as a `media_id` to the YAML for this audio file.
+
+```YAML
+- id: "cva_podcast"
+  src: /screenshots/cva_podcast.png
```

Comment:
I think this might use `poster` too, in place of `src`, or it's possible that neither do anything at the moment.